draft-private-access-tokens.md

```diff
@@ -471,16 +478,15 @@ For each tuple of (Client, ANON_ORIGIN_ID, policy window), the Mediator maintain
 following state:
 
 - A counter of successful tokens issued
-- Whether or not a previous request was rejected by the Issuer  
+- Whether or not a previous request was rejected by the Issuer
 - The last received ANON_ISSUER_ORIGIN_ID value for this ANON_ORIGIN_ID, if any
 
 ### Issuer State {#issuer-state}
 
 Issuers maintain a stable ISSUER_SECRET that they use in calculating values returned
 to the Mediator. If this value changes, it will open up a possibility for Clients to
 request extra tokens for an Origin without being limited, within a policy window.
-
-TODO: Chris to describe how ISSUER_SECRET should be calculated
+ISSUER_SECRET
```

Comment:
@chris-wood was this intended to describe how ISSUER_SECRET is calculated?

Reply:
YEp :) Will push